documentation/asset-tracker/topics/how-assettracker-works.md

```diff
@@ -0,0 +1,190 @@
+(how-assettracker-works-doc)=
+
+# How AssetTracker works
+
+AssetTracker builds on Hyperview's capabilities by adding specialized hardware and software to achieve automated Rack U-level tracking across one or more data centers.
+
+## Terminology
+
+- **Gateway**, the is the devices that handles communication between the hardware and the data collector.
```

Comment:
Should be "this is the device..." instead of "the is the devices...".

Reply:
Fixed, thanks!